README.md

```diff
@@ -11,6 +11,21 @@ To build the Digirati Taxonomy Manager, use the included gradle wrapper:
 ./gradlew build
 ```
 
+Once you have built the project locally you can use docker-compose to get an environment up, with a database and sample configuration:
+```
```

Comment:
[MD031](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md031---fenced-code-blocks-should-be-surrounded-by-blank-lines), there should be blank lines around fences.

Same for the other fences that have been added which don't have blank lines.